src/digest-stream.js

```diff
@@ -0,0 +1,28 @@
+import stream from 'stream';
```

Comment:
This class is not intended to be used outside of this library.  Given that, does adding a dependency on that package really make sense?